nullaway/src/main/java/com/uber/nullaway/DummyOptionsConfig.java

```diff
@@ -51,7 +52,15 @@
           + FL_ANNOTATED_PACKAGES
           + "=[...] flag.";
 
-  public DummyOptionsConfig() {}
```

Comment:
Why is this line deleted?

Reply:
It was deleted by mistake, thank you for noting it. Brought it back 41833d9ea841af253c7fc3346394d2d8ba378da2.